### PR TITLE
Get rid of jQuery dependency

### PR DESCRIPTION
**Pull request by dxvladislavvolkov**
The jQuery dependency has been removed [#46](https://github.com/DevExpress/devextreme-angular/issues/46)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,9 +16,6 @@
   },
   "author": "Developer Express Inc.",
   "license": "MIT",
-  "dependencies": {
-    "jquery": "^2.0.0 || ^3.0.0"
```
Comment (kvet) [inline package.json]: What about types for jquery?

File: src/core/nested-option.ts [hunk 2]
```diff
@@ -144,40 +157,27 @@ export function extractTemplate(option: IOptionWithTemplate, element: ElementRef
...
-
     option.template = {
         render: (renderData) => {
-            let $result = $(element.nativeElement).addClass(DX_TEMPLATE_WRAPPER_CLASS);
+            let result = element.nativeElement;
+            result.classList.add(DX_TEMPLATE_WRAPPER_CLASS);
 
             if (renderData.container) {
-                let container = renderData.container.get(0);
+                let container = renderData.container.get ? renderData.container.get(0) : renderData.container;
```
Comment (kvet) [inline src/core/nested-option.ts]: Maybe we should extract a helper method with an understandable name?

File: src/core/template.ts [hunk 3]
```diff
@@ -44,22 +42,31 @@ export class DxTemplateDirective {
             '$implicit': renderData.model,
             index: renderData.index
         });
+        let container = renderData.container.get ? renderData.container.get(0) : renderData.container;
```
Comment (kvet) [inline src/core/template.ts]: The same here

File: tests/src/ui/custom-value-accessor-implementation.spec.ts [hunk 4]
```diff
@@ -71,12 +71,12 @@ describe('DxTextBox value accessor', () => {
         fixture.componentInstance.formControl.disable();
         fixture.detectChanges();
 
-        expect(instance.option('disabled')).toBe(true);
+        expect(instance['option']('disabled')).toBe(true);
```
Comment (kvet) [inline tests/src/ui/custom-value-accessor-implementation.spec.ts]: Strange change

File: README.md [hunk 5]
```diff
@@ -137,6 +139,13 @@ npm start
 
 Navigate to [http://127.0.0.1:8875/examples/](http://127.0.0.1:8875/examples/) in the opened browser window. Explore the **examples** folder of this repository for the examples source code.
 
+### <a name="jquery integration"></a>Include jQuery integration ###
+DevExtreme has removed the jQuery dependence in version 17.2. It means that our widgets work without jQuery elements.If you need to use jQuery, you can include the jQuery integration as described below:
```
Comment (kvet) [inline README.md]: Missed space after dot.

File: src/core/nested-option.ts (hunk 2 above)
Comment (Aden-git) [reply src/core/nested-option.ts]: Is it not a temporary solution, that already should be changed to DOM elements?

Comment (ovchinnikov) [reply src/core/nested-option.ts]: No, it is not. You can add the jQuery integration into your application and API of DevExtreme widgets will change

File: package.json (hunk 1 above)
Comment (Aden-git) [reply package.json]: Shouldn't we update a `devextreme` peerDependency?

File: package.json [hunk 6]
```diff
@@ -16,9 +16,6 @@
...
-  },
   "peerDependencies": {
     "devextreme": "~17.2.1-0",
```
Comment (Aden-git) [inline package.json]: Shouldn't we update a `devextreme` peerDependency?

File: src/core/utils.ts [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+            } else {
+                element.className = element.className ? element.className + ' ' + name : name;
+            }
+        }
+    };
+
+    public static getElement(element: any) {
+         return element.get ? element.get(0) : element;
+    };
+}
```
Comment (kvet) [inline src/core/utils.ts]: Defining a class with static members is not a JavaScript way. Export static methods directly.

**Pull request merged**